mdpow/tests/test_automated_dihedral_analysis.py

```diff
@@ -0,0 +1,215 @@
+import os
+import sys
+import yaml
+import pybol
+import pytest
+import pathlib
+
+import scipy
+import numpy as np
+import pandas as pd
+
+import rdkit
+from rdkit import Chem
+
+from numpy.testing import assert_almost_equal
+from scipy.stats import circvar, circmean
+
+from . import RESOURCES
+
+import py.path
+
+from ..workflows import dihedrals
+
+from pkg_resources import resource_filename
+
+# NEW TESTING DATA NEEDS TO BE GENERATED AFTER
+# REDUCING TESTING DATASET
+
+# Are any additional tests needed?
+# Top level?
+
+RESOURCES = pathlib.PurePath(resource_filename(__name__, 'testing_resources'))
+MANIFEST = RESOURCES / "manifest.yml"
+
+@pytest.fixture(scope="function")
+def molname_workflows_directory(tmp_path, molname='SM25'):
+    m = pybol.Manifest(str(MANIFEST))
+    m.assemble('workflows', tmp_path)
+    return tmp_path / molname
+
+class TestAutomatedDihedralAnalysis(object):
+
+    @pytest.fixture(scope="function")
+    def SM25_tmp_dir(self, molname_workflows_directory):
+        dirname = molname_workflows_directory
+        return dirname
+
+    @pytest.fixture(scope="function")
+    def gen_data(self, SM25_tmp_dir):
```

Comment:
Can you split gen_data into two fixtures
- `dihedral_data`: produces `atom_group_indices, df, df_aug` (runs analysis) and is used when we want to look at the analysis output
- `atom_indices`: produces `atom_group_indices, atom_group_indices_alt` (only does the initial dihedral_indices) and is used when checking indices

Reply:
Responding to question in comment above, I think all workflows related testing data in this PR is 14MB in total.